### ISSUE TITLE
[5.x]: Coupon discount  - unable to disable `Ignore promotional prices..`

### ISSUE THREAD

**Issue by mikesnoeren** [craftcms/commerce#3573]
### Description
When trying to disable `Ignore promotional prices when this discount is applied to matching line items` on any discount, it gets enabled again after saving discount.

### Steps to reproduce

1. Go to any discount
2. disable `Ignore promotional prices when this discount is applied to matching line items`
3. Save the discount
4. Check the lightswitch, it will be enabled again.

### Craft CMS version

5.2.5

### Craft Commerce version

5.0.11.1

### Installed plugins and versions

-

**Comment by lukeholder** [comment]
This has been fixed for the next release which will be out shortly.

To get the fix early, change your `craftcms/commerce` requirement in composer.json to:

```json
"require": {
  "craftcms/commerce": "5.x-dev#524593c2d54349d583760f6f3fcd45f86712ca5c as 5.0.12.2",
  "...": "..."
}
```

Then run `composer update`.

We will update this ticket once the release it out.

**Comment by lukeholder** [comment]
This was released in Craft Commerce 5.0.13  Thanks!
